app/addons/documents/index-results/components/results/TableRow.js

```diff
@@ -64,11 +65,14 @@ export default class TableRow extends React.Component {
   maybeGetCheckboxCell (el, i) {
     return (
       <td className="tableview-checkbox-cell" key={"tableview-checkbox-cell-" + i}>
-        {el.isDeletable ? <input
-          id={"checkbox-" + this.props.docIdentifier}
-          checked={this.props.isSelected}
-          type="checkbox"
-          onChange={this.onChange.bind(this)} /> : null}
+        {el.isDeletable ? <React.Fragment>
+          <Form.Check
+            id={"checkbox-" + this.props.docIdentifier}
+            checked={this.props.isSelected}
+            type="checkbox"
+            onChange={this.onChange.bind(this)}
+          />
+        </React.Fragment> : null}
```

Comment:
You can remove `<React.Fragment>` since there's only 1 element inside it.

Reply:
Addressed @Antonio-Maranhao